lib/client/index.js

```diff
@@ -52,74 +52,58 @@ module.exports = ({ port = null, config = {}, filters = {} }) => {
       config.brokerClientValidationTimeoutMs || 5000;
     const isJsonResponse = !config.brokerClientValidationJsonDisabled;
 
-    const data = {
-      brokerClientValidationUrl: logger.sanitise(
-        config.brokerClientValidationUrl,
-      ),
-      brokerClientValidationMethod,
-      brokerClientValidationTimeoutMs,
-    };
-
-    const validationRequestHeaders = {
-      'user-agent': 'Snyk Broker client ' + version,
-    };
-
     // set auth header according to config
-    if (config.brokerClientValidationAuthorizationHeader) {
-      validationRequestHeaders.authorization =
-        config.brokerClientValidationAuthorizationHeader;
+    function credsFromHeader(s) {
```

Comment:
Is there a reason the `credsFromHeader` function being defined in the scope of the handler for the `/systemcheck` endpoint?
Extracting it would make the handler less bloated.

Reply:
If I'm extracting the bit below to a function (which I am), then it definitely does.